### PR TITLE
feat: Add eXOF to Valora Supported Token List

### PR DESCRIPTION
**Pull request by SergeWilfried**

### CODE REVIEW

**Comment by jeanregisser** [comment]
Thanks for the contribution but we already added about a month ago: https://github.com/valora-inc/address-metadata/commit/d6ff051484578b1e9af773866242103e4ee07904

Could you update the existing ones with the additional info you have? 🙏

**Comment by SergeWilfried** [comment]
> jeanregisser

Sure

**Comment by SergeWilfried** [comment]
@jeanregisser done !

File: src/data/mainnet/celo-tokens-info.json [hunk 1]
```diff
@@ -653,7 +654,10 @@
     "imageUrl": "https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/eXOF.png",
     "isCoreToken": true,
     "name": "ECO CFA",
-    "symbol": "eXOF"
+    "symbol": "eXOF",
+    "isStableCoin": true,
+    "isCashOutEligible": true,
```
Comment (jh2oman) [inline src/data/mainnet/celo-tokens-info.json:659]: Can you remove these `isCashOutEligible` fields? We actually need some work in the wallet code first to be able to support a new cash out currency. 

Btw, what cash out providers support eXOF?

Comment (jh2oman) [inline src/data/mainnet/celo-tokens-info.json:659]: Can you remove these isCashOutEligible fields? We actually need some work in the wallet code first to be able to support a new cash out currency.

Btw, what cash out providers support eXOF?

File: src/data/mainnet/celo-tokens-info.json [hunk 2]
```diff
@@ -653,7 +654,10 @@
...
+    "infoUrl": "https://www.dunia.africa"
```
Comment (jh2oman) [inline src/data/mainnet/celo-tokens-info.json:660]: I can't find any info on eXOF when i goto this link. Can you provide a more specific url?

**Review by jh2oman — CHANGES_REQUESTED** [review]

File: src/data/mainnet/celo-tokens-info.json (hunk 2 above)
Comment (SergeWilfried) [reply src/data/mainnet/celo-tokens-info.json]: The website is going through an design update at the moment. I will update this once the new website is ready.

**Comment by silasbw** [comment]
Closing due to inactivity -- feel free to update and re-open if you still want to get this added.

**Pull request closed without merging**